LoopUI/Views/LoopCompletionHUDView.swift

```diff
@@ -160,3 +160,26 @@ public final class LoopCompletionHUDView: BaseHUDView {
         assertTimer()
     }
 }
+
+extension LoopCompletionHUDView {
+    public var loopCompletionMessage: (title: String, message: String)? {
+        switch freshness {
+        case .fresh:
+            if loopStateView.open {
+                return (title: NSLocalizedString("Closed Loop OFF", comment: "title of green open loop OFF message"),
+                        message: NSLocalizedString("\nTidepool Loop is operating with Closed Loop in the OFF position. Your pump and CGM will continue operating, but your basal insulin will not adjust automatically.\n\nTap Settings to toggle Closed Loop ON if you wish for the app to automate your insulin.", comment: "Green closed loop OFF message"))
+            } else {
+                return (title: NSLocalizedString("Closed Loop ON", comment: "title of green closed loop ON message"),
+                        message: NSLocalizedString("\nTidepool Loop is operating with Closed Loop in the ON position. Your last loop was successful within the last 5 minutes.", comment: "Green closed loop ON message"))
+            }
+        case .aging:
+            return (title: NSLocalizedString("Loop Failure", comment: "title of yellow loop message"),
+                    message: NSLocalizedString("\nTidepool Loop has not completed a loop successfully in the past 5-15 minutes.\n\nTap your CGM and insulin pump status icons for more information. Tidepool Loop will continue trying to complete a loop, but watch for potential communication issues with your pump and CGM.", comment: "Yellow loop message"))
+        case .stale:
+            return (title: NSLocalizedString("Loop Failure", comment: "title of red loop message"),
+                    message: NSLocalizedString("\nTidepool Loop has not completed a loop successfully in over 15 minutes.\n\nTap your CGM and insulin pump status icons for more information. Tidepool Loop will continue trying to complete a loop, but check for potential communication issues with your pump and CGM.", comment: "Red loop message"))
+        default:
+            return nil
```

Comment:
[nit] Seems to me we should show something for the `.unknown` case. When can `.unknown` happen? Looks like when either:
- a successful loop has not yet completed at least once since *this* app launch
- last successful loop was > 12 hours ago

Seems like there needs to be two additional messages.  One for "loop has not yet successfully completed at least once since app start" and "last loop is older than 12 hours ago".

Reply:
@darinkrauss The default state for `LoopCompletionFreshness` in the LoopCompletionHUDView is now `.stale`. Also, the LoopCompletionFreshness of `.unknown` should never occur in Loop. Makes me question if we should keep it around?

Reply:
If `.unknown` is no longer used, then I'd recommend getting rid of it. Otherwise it is cruft and may cause confusion later on.